browser.js

```diff
@@ -1,27 +1,7 @@
-var createHmac = require('create-hmac')
-var MAX_ALLOC = Math.pow(2, 30) - 1 // default in iojs
-
-exports.pbkdf2 = pbkdf2
-function pbkdf2 (password, salt, iterations, keylen, digest, callback) {
-  if (typeof digest === 'function') {
-    callback = digest
-    digest = undefined
-  }
-
-  checkParameters(iterations, keylen)
-
-  if (typeof callback !== 'function') {
-    throw new Error('No callback provided to pbkdf2')
-  }
-
-  setTimeout(function () {
-    var result = pbkdf2Sync(password, salt, iterations, keylen, digest)
-
-    callback(null, result)
-  })
-}
+exports._createHmac = require('create-hmac/browser')
```

Comment:
why is this exported? can't it be imported again?  Also why does this need to be the /browser, those are mainly intended for internal testing purposes and could totally change in the future (but at the moment require('create-hmac') and  require('create-hmac/browser') resolve to be the same in the browser).